Server/Ebenezer/User.cpp

```diff
@@ -12070,6 +12074,77 @@ bool CUser::CheckClass(short class1, short class2, short class3, short class4, s
 	return false;
 }
 
+bool CUser::CheckPromotionEligible()
+{
+	CNpc* npc = m_pMain->m_NpcMap.GetData(m_sEventNid);
+	if (npc == nullptr)
+		return false;
+
+	short tempClass = -1;
```

Comment:
This is now unused - when the existing merge conflicts are handled, this will trigger a warning which will fail the build.